tests/conftest.py

```diff
@@ -211,24 +209,16 @@ def context(vcr, unique, unique_lower, freezer):
 @pytest.fixture(scope="session")
 def record_mode(request):
     """Manage compatibility with DD client libraries."""
-    return {
-        "false": "none",
-        "true": "rewrite",
-        "none": "new_episodes",
-    }[os.getenv("RECORD", "false").lower()]
+    return {"false": "none", "true": "rewrite", "none": "new_episodes",}[
+        os.getenv("RECORD", "false").lower()
+    ]
 
 
 def _disable_recording():
     """Disable VCR.py integration."""
     return os.getenv("RECORD", "false").lower() == "none"
 
 
-@pytest.fixture(scope="session")
```

Comment:
Why are we removing this ?

Reply:
It's not used anywhere, and I'm not sure what it does.

Reply:
It's a redefinition of a builtin fixture: https://github.com/kiwicom/pytest-recording/blob/master/src/pytest_recording/plugin.py#L74

Reply:
Do you think we need it?

Reply:
Good catch!